### ISSUE TITLE
ImportError: cannot import name '_l2_norm' from 'gensim.models.keyedvectors

### ISSUE THREAD

**Issue by pratikkotian04** [oborchers/Fast_Sentence_Embeddings#64]
ImportError                               Traceback (most recent call last)

<ipython-input-13-30726af9f93b> in <module>()
----> 1 from fse import Vectors, Average, IndexedList
      2 vecs = Vectors.from_pretrained("fasttext-wiki-news-subwords-300")
      3 model = Average(vecs)

3 frames

/usr/local/lib/python3.7/dist-packages/fse/models/base_s2v.py in <module>()
     40 
     41 from gensim.models.base_any2vec import BaseWordEmbeddingsModel
---> 42 from gensim.models.keyedvectors import BaseKeyedVectors, FastTextKeyedVectors, _l2_norm
     43 from gensim.utils import SaveLoad
     44 from gensim.matutils import zeros_aligned

ImportError: cannot import name '_l2_norm' from 'gensim.models.keyedvectors' (/usr/local/lib/python3.7/dist-packages/gensim/models/keyedvectors.py)

**Comment by AleMuzzi** [comment]
Hi, I don't know which version of Gensim you are using but I can tell you that in versions >=4 the `_l2_norm` function has been removed from `keyedvectors.py`.
You can either choose to

- edit your locally downloaded version of Gensim replacing `_l2_norm` with `linalg.norm` after adding linalg to the numpy import in the file's imports section
- update your fse with the porting I just made for Gensim >=4 (which also solves this problem) by executing the command
```
pip install -U git+https://github.com/AleMuzzi/Fast_Sentence_Embeddings
```
 works both as an upgrade and as a new version, pull request is waiting for approval (https://github.com/oborchers/Fast_Sentence_Embeddings/pull/65)

**Comment by oborchers** [comment]
Should be fixed in `v1.0.0`
